doc/rtrtr.1

```diff
@@ -304,9 +305,18 @@ colon. IPv6 address should be enclosed in square brackets.
 .B unit
 A string value specifying the name of the unit that provides the data
 set for the RTR target to offer.
+.TP
+.B history\-size
+An integer value specifying the number of diffs the target should keep
+in order to process RTR serial queries, i.e., the number of updates to
```

Comment:
Remove the comma after `i.e.` in `i.e., the number of updates`.

Reply:
A quick google suggests that the comma after i.e. and e.g. is an American English thing. Since I tend to use that in my comments, I’d like to keep it.